source/2019-11-29-the-ember-times-issue-126.md

```diff
@@ -65,13 +66,17 @@ If you can contribute to everyone's learning today, we ask that you help **updat
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Ember RealWorld App is Feature Complete 🌠](https://github.com/gothinkster/ember-realworld)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+[RealWorld](https://github.com/gothinkster/realworld), "The mother of all demo apps", is an Open Source initiative to showcase a full stack app build using any frontend or backend framework/library. It provides a spec for both the API and the frontend. The idea is to show how a "real world" app can be built using different frameworks/libraries.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The Ember Realworld implementation is finally complete! It was started over two years ago by [@Alonski](https://github.com/Alonski) and after all this time is finally feature complete. A large number of contributors helped out along the way: [@jonnii](https://github.com/jonnii), [@alexlafroscia](https://github.com/alexlafroscia), [@delusioninabox](https://github.com/delusioninabox), [@patocallaghan](https://github.com/patocallaghan), [@GCheung55](https://github.com/GCheung55), [mansona](https://github.com/mansona), [@johpol](https://github.com/johpol)
+
+The implementation showcases a class Ember.js app using the most idiomatic Ember patterns to serve as an example for the wider community. Feel free to look through the codebase and even offer suggestions, PRs or comments regarding how the app is built. There is always room for improvements. We are definitely looking for performance improvements so if thats your cup of tea ☕ please contribute.
+
+The RealWorld project has a rating system based on GitHub stars. Currently Ember RealWorld has 58 stars while the React/Redux implementation has over 3000. Can we band together and show that Ember is actually used in the "RealWorld"? Go and vote with your stars 🌟!
```

Comment:
I really like that you convey the idea that the community is already showing its support for the project with almost 60 stars, but I personally think it'd be nice to focus on what this specific project could be without the direct comparison to React - which I believe would make it sound less competitive and even more agreeable.

What do you think about something along the lines of:

```suggestion
The RealWorld project has a rating system based on GitHub stars. Currently Ember RealWorld has been starred by community members 58 times, but we believe there could be so many more! To make the repo reflect the real world usage of Ember on the web, go and vote with your stars 🌟!
```

Let me know what you think!